released/packages/coq-vst-32/coq-vst-32.2.11.1/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "Verified Software Toolchain"
+description: "The software toolchain includes static analyzers to check assertions about your program; optimizing compilers to translate your program to machine language; operating systems and libraries to supply context for your program. The Verified Software Toolchain project assures with machine-checked proofs that the assertions claimed at the top of the toolchain really hold in the machine-language program, running in the operating-system context."
+authors: [
+  "Andrew W. Appel"
+  "Lennart Beringer"
+  "Josiah Dodds"
+  "Qinxiang Cao"
+  "Aquinas Hobor"
+  "Gordon Stewart"
+  "Qinshi Wang"
+  "Sandrine Blazy"
+  "Santiago Cuellar"
+  "Robert Dockins"
+  "Nick Giannarakis"
+  "Samuel Gruetter"
+  "Jean-Marie Madiot"
+]
+maintainer: "VST team"
+homepage: "http://vst.cs.princeton.edu/"
+dev-repo: "git+https://github.com/PrincetonUniversity/VST.git"
+bug-reports: "https://github.com/PrincetonUniversity/VST/issues"
+license: "BSD-2-Clause"
+
+build: [
+  [make "-j%{jobs}%" "vst" "IGNORECOQVERSION=true" "ZLIST=platform" "BITSIZE=32"]
+]
+install: [
+  [make "install" "IGNORECOQVERSION=true" "ZLIST=platform" "BITSIZE=32"]
+]
+run-test: [
+  [make "-j%{jobs}%" "test" "IGNORECOQVERSION=true" "ZLIST=platform" "BITSIZE=32"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.14" & < "8.17~"}
+  "coq-compcert-32" {= "3.11"}
+  "coq-vst-zlist" {= "2.11"}
+  "coq-flocq" {>= "4.1.0"}
+]
+conflicts: [
+  "coq-vst"
+]
```

Comment:
Did this really conflict with `coq-vst`? If I remember correctly, you said that both `coq-vst` and `coq-vst-32` could be installed simultaneously, thanks to the Makefile in VST putting files in different places.

Reply:
Indeed I copied this from the previous version - it should not be conflicting but it is hard to test these days since 32 bit is neither supported on Mac nor on Windows and on Linux requires the 32 bit cross toolkit.